src/footer/footer.macros.html

```diff
@@ -7,53 +7,42 @@
 {#
   * Renders a standard or simplified footer
 #}
-{% macro footer(simple=false, class=null, id=null, phone_number='888–498–0003') %}
+{% macro footer(data, simple=false, class=null, id=null) %}
 <footer class="footer{% if simple %} footer--simple{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
   <div class="footer-top r-inner">
     {{ caller() }}
   </div>
   <div class="footer-bottom">
```

Comment:
What's a better class to replace `footer-bottom`? I like the feel of the the `site-` classes to describe the content, but it now feels disconnected from its parent class.